ReadMe.md

```diff
@@ -1,11 +1,14 @@
-[![Kotlin](https://img.shields.io/badge/kotlin-1.0.0-blue.svg)](https://kotlinlang.org) [![Kotlin Slack](https://img.shields.io/badge/chat-kotlin%20slack-orange.svg)](https://kotlinslackin.herokuapp.com)
+[![Kotlin Experimental](https://kotl.in/badges/experimental.svg)](https://kotlinlang.org/docs/components-stability.html)
+[![JetBrains incubator project](https://jb.gg/badges/incubator.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
+[![Kotlin](https://img.shields.io/badge/kotlin-1.0.0-blue.svg)](https://kotlinlang.org) 
+[![Kotlin Slack](https://img.shields.io/badge/chat-kotlin%20slack-orange.svg)](https://kotlinslackin.herokuapp.com)
 
 
 # kotlinx.reflect.lite
 
 This library provides an API to introspect Kotlin symbols at runtime. Its main feature is the small size which makes it possible to use it on memory-constrained devices.
 
-[ ![Download](https://api.bintray.com/packages/kotlin/kotlinx.reflect.lite/kotlinx.reflect.lite/images/download.svg) ](https://bintray.com/kotlin/kotlinx.reflect.lite/kotlinx.reflect.lite/_latestVersion)
+[Download](https://bintray.com/kotlin/kotlinx.reflect.lite/kotlinx.reflect.lite/_latestVersion)
```

Comment:
I guess this link should be removed completely, since bintray is now dead. And downloading library from the repo doesn't make much sense anyway.

Reply:
We'll clean it up separately as soon as the first prototype is ready, thanks